docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -1,159 +1,191 @@
+install and test these capabilities **in a test environment**. You might run
+into problems and need to modify your setup to get this feature running. Please
+do not enable or use {ingest-management} in a production environment.
+// end::experimental-warning[]
+
+//REVIEWERS: We have several different statements like this one that say basically the same
+//thing but use slightly different words. Let's create a standard description that I
+//can include when we need to add this statement. Let me know if what I have here ^^
+//is good.
 
-//TODO: Add cloud info
+For feedback and questions, please contact us in the
+https://ela.st/ingest-manager-feedback[discuss forum].
 
-IMPORTANT: Since this is an experimental release, it should only be used in a dedicated test environment
-that can be deleted when you are done. Please do not enable or use {ingest-management} in a production
-environment.
+//TODO: Add link to limitations topic after it is merged. 
 
 [float]
 [[ingest-manager-prereqs]]
 == Prerequisites
 
-If you want to use {ingest-management}, the following prerequisites have to be met in the {es} configuration:
+To use this **experimental** release of {ingest-management}, you need:
 
-- {es} security has to be enabled by setting `xpack.security.enabled` to `true`.
-- API keys have to be enabled in {es} by setting `xpack.security.authc.api_key.enabled` to `true`.
+* An {es} cluster and {kib} (version 7.8) with a basic license.
++
+You can use our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]
+on {ecloud}, or {stack-gs}/get-started-elastic-stack.html[install the {stack} on
+your own hardware].
 
-If you use our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on
-{ecloud}, these settings will already be enabled.
+* An {es} configuration with the following capabilities enabled. If you use our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+these settings are already enabled.
++
+- {ref}/configuring-security.html[{es} security]. The `xpack.security.enabled`
+setting must be `true`.
+- {ref}/security-settings.html#api-key-service-settings[API key service]. The
+`xpack.security.authc.api_key.enabled` setting must be `true`.
 
-In this experimental release, {ingest-management} is only available to users with the superuser role. See
-{ref}/built-in-roles.html[Built-in roles].
+* A {kib} configuration with the following capabilities enabled:
++
+- {kibana-ref}/using-kibana-with-security.html[{kib} security]. The `xpack.security.enabled`
+setting must be `true`.
+- {kibana-ref}/configuring-tls.html[TLS] or the
+`pack.ingestManager.fleet.tlsCheckDisabled` setting must be `true`.
 
-For feedback and questions, please contact us https://ela.st/ingest-manager-feedback[here]. 
+//REVIEWERS: I've added the Kibana requirements because they weren't stated. Let
+//me know if you'd rather that I leave those out. I personally have concerns
+//about telling users to disable TLS checks.
```

Comment:
Cc @nchaulet about the TLS checks and when exactly they need to be disabled. Maybe adding more reasoning why this is necessary in some situations helps? (If not, please ignore.)

Reply:
Thanks! If necessary, I can open a follow-up PR after this PR is merged.

Reply:
The scenario where the user wants to disable the TLS check, is if Kibana is behind a proxy that terminate the ssl connection.